playground/.ladle/config.mjs

```diff
@@ -4,7 +4,7 @@ export default {
   addons: {
     width: {
       options: {
-        xsmall: 375,
+        xsmall: 320,
```

Comment:
why do we have this change!? do we have any discussion on why should we have xsmall 320!?